init.el

```diff
@@ -1,12 +1,15 @@
 ;;; init.el -*- lexical-binding: t; -*-
 
+;; Don't save customize variables in `init.el', save them in
+;; "emacs-custom.el" instead. Don't bail out if file doesn't exist.
+(setq custom-file (locate-user-emacs-file "emacs-custom.el"))
+(load custom-file 'noerror 'nomessage)
+
```

Comment:
This change seems only slightly related to the other changes and is also opinionated. I'd move that one in another pull request, to be honest.